README.md

```diff
@@ -21,18 +21,35 @@ For a more in-depth overview of openCryptoki, please refer to manual
 Does not work with TPM version 2.0. We plan to remove the TPM token in a future 
 openCryptoki release or version.
 
+AIX only supports the CCA and software tokens, and both are enabled in a default build configuration. Other tokens are unsupported and cannot be force-enabled through the `configure` utility.
+
 ## REQUIREMENTS:
 
+### Common
+Building opencryptoki needs the following utilities.
+- flex
+- bison
+- make
+- autoconf
+- automake
+- pkg-config
+- libtool
+- m4
+- openldap-devel
+
```

Comment:
On linux, you also need libcap-devel and systemd-devel. 
Are there any additional libraries on AIX ?

Reply:
Nope, there are no AIX-native libraries that are used. Everything in use comes bundled within `libc.a` and `libs.a`, which are implicitly included.